myia/composite.py

```diff
@@ -617,10 +617,10 @@ def generate_graph(self, args):
 
         gnext = Graph()
         gnext.debug.name = self._decorate_name('lm_body')
-        gnext.flags['ignore_values'] = True
+        gnext.set_flags('reference', 'ignore_values')
```

Comment:
Are those really reference graphs?

Reply:
The graph doesn't have the core flag because we don't want to inline it.

Reply:
Ok